City-of-Helsinki/helsinki-paatokset
# PP-207: Search implementation

Included in this commit:

Search API configuration
Indices for decisions / policymakers
Views for the two searches
Forms for the two searches + changes to paragraph with search form embedded

To test:

- `composer install`
- `make drush-cim`
- `make drush-updb`
- Run the `paatokset_agenda_items` -migration
- Go to search api settings (`admin/config/search/search-api`). Visit each index and run the ndexing process
- Check that ``/search-decisions` and `/search-policymakers` routes work for you
- Compare these view to the ones in sketches
